ajax/createLabelAjax.php

```diff
@@ -0,0 +1,218 @@
+<?php
+
+/*
+ * 2007-2013 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ *  @author PrestaShop SA <contact@prestashop.com>
+ *  @copyright  2007-2013 PrestaShop SA
+ *  @version  Release: 0.4.4
+ *  @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ *  International Registered Trademark & Property of PrestaShop SA
+ */
+
+$useSSL = false;
+if(isset($_SERVER["HTTPS"]) && in_array($_SERVER["HTTPS"], array("on", "ON"))){ $useSSL = true; }
+// die("useSSL: ".(int)$useSSL);
+include('../../../config/config.inc.php');
+include('../../../init.php');
+
+if (!class_exists('SeurLib')) include(_PS_MODULE_DIR_ . 'seur/classes/SeurLib.php');
+if (!class_exists('Pickup')) include(_PS_MODULE_DIR_ . 'seur/classes/Pickup.php');
+
+$order_data = (array) $_GET;
+
+if (Tools::getValue('token') != Tools::getAdminToken('AdminOrders' . (int) (Tab::getIdFromClassName('AdminOrders')) . (int) (Tools::getValue('id_employee'))))
+    Tools::display404Error();
```

Comment:
Please note this function is deprecated from PS 1.5.5.0. You should use `Controller::getController('PageNotFoundController')->run();` as written in the comment.